examples/Default-Reproduction.v1.yaml

```diff
@@ -0,0 +1,663 @@
+          schema:
+            type: string
+            example: seattle-office
+            default: all
+          in: query
+          name: spaceid
+          description: >-
+            location associated with the devices (see `'My Spaces'` for more
+            information)
+          allowEmptyValue: true
+        - &ref_5
+          schema:
+            type: string
+            enum:
+              - asc
+              - desc
+            default: desc
+            example: asc
+          in: query
+          name: sort
+          allowEmptyValue: true
+          description: will sort objects returned from the request
+        - &ref_6
+          schema:
+            type: integer
+            default: 10
+            format: int32
+            example: 15
+            maximum: 25
+            minimum: 1
+          in: query
+          name: limit
+          allowEmptyValue: true
+          description: specify the number of results returned from the request
+        - &ref_3
+          schema:
+            type: string
+            default: all
+            example: 123abc
+          in: header
+          name: Deviceid
+          description: >-
+            unique identifiers for devices registered on the service cloud, when
+            empty it defaults to `'all'`
+          allowEmptyValue: true
+          required: true
+    post:
+      summary: newDevice
+      tags:
+        - Devices
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema: *ref_0
+              examples:
+                Created:
+                  value:
+                    code: 201
+                    message: Created
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema: *ref_0
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema: *ref_0
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema: *ref_0
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema: *ref_0
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema: *ref_0
+        '502':
+          description: Bad Gateway
+          content:
+            application/json:
+              schema: *ref_0
+        '504':
+          description: Gateway Timeout
+          content:
+            application/json:
+              schema: *ref_0
+        default:
+          description: Error
+          content:
+            application/json:
+              schema: *ref_0
+      operationId: post-api-devices
+      description: >-
+        <span style="font-size: 1.5em; color: #5D535E;">
+          <i class="fas fa-robot fa-xs"></i> newDevice
+        </span>
```

Comment:
@chohmann Leftover file?

Reply:
yep! Accidentally hit stage all in Tower. thanks. Removed cf79216